velox/exec/tests/HashJoinTest.cpp

```diff
@@ -150,95 +312,199 @@ class HashJoinTest : public HiveConnectorTestBase {
     }
     return count;
   }
+
+  const int numDrivers_;
+
+  VectorFuzzer::Options fuzzerOpts_;
+  VectorFuzzer fuzzer_;
+
+  // The default left and right table types used for test.
+  RowTypePtr leftType_;
+  RowTypePtr rightType_;
 };
 
-TEST_F(HashJoinTest, bigintArray) {
+class MultiThreadedHashJoinTest
+    : public HashJoinTest,
+      public testing::WithParamInterface<TestParam> {
+ public:
+  MultiThreadedHashJoinTest() : HashJoinTest(GetParam()) {}
+
+  static std::vector<TestParam> getTestParams() {
+    return std::vector<TestParam>({TestParam{1}, TestParam{4}});
+  }
+};
+
+TEST_P(MultiThreadedHashJoinTest, bigintArray) {
   testJoin(
       {BIGINT()},
-      1,
-      16000,
-      15000,
+      numDrivers_,
+      1600,
+      3,
+      1500,
+      3,
       "SELECT t_k0, t_data, u_k0, u_data FROM "
       "  t, u "
       "  WHERE t_k0 = u_k0");
 }
 
-TEST_F(HashJoinTest, bigintArrayParallel) {
```

Comment:
Is it okay to remove this test?

Reply:
Yeah, it is now covered by the multithreaded test mode.